examples/pytorch/llm/scripts/qwen_72b_chat/torchacc/lora_fsdp_sft.sh

```diff
@@ -0,0 +1,27 @@
+export USE_TORCHACC=1
+export XLA_FLAGS='--xla_gpu_enable_latency_hiding_scheduler=true --xla_enable_pt_all_reduce_optimizer=false --xla_gpu_enable_async_all_gather=true --xla_gpu_enable_async_reduce_scatter=true --xla_multiheap_size_constraint_per_heap=4294967296 --xla_disable_hlo_passes=rematerialization,all-gather-combiner,all-reduce-combiner,reduce-scatter-combiner'
+export ACC_FLASH_ATTN=1
+export LOW_CPU_MEM_USAGE=1
+export XLA_IR_SHAPE_CACHE_SIZE=100000000
+export XLA_ALLOCATOR_FRACTION=0.97
+
+MASTER_ADDR=127.0.0.1 \
+MASTER_PORT=9010 \
+NPROC_PER_NODE=2 \
```

Comment:
Can you provide an example that demonstrates the advantages of torchacc? and create a related introductory document in LLM to attract users to use it.